https://github.com/Moonbase59/loudgain/compare/508b6fd63cfc8b3db67a0eb19d87a036543325f4...d6ae9c08ca0e88cc3caebd499185c26f9c1420da
(but stripped docs/loudgain.1.html diff since they won't be installed to the system.)
From 31f46fcc55b56618730e269640d671ce65891612 Mon Sep 17 00:00:00 2001
From: Moonbase59 <moonbase@quantentunnel.de>
Date: Fri, 6 Sep 2019 18:18:33 +0200
Subject: [PATCH 1/2] README: Add APE to introduction

---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index e77fb72..1f82fa2 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # loudgain
 
-**loudgain** is a versatile ReplayGain 2.0 loudness normalizer, based on the EBU R128/ITU BS.1770 standard (-18 LUFS) and supports FLAC/Ogg/MP2/MP3/MP4/M4A/ALAC/Opus/ASF/WMA/WAV/WavPack/AIFF audio files. It uses the well-known `mp3gain` commandline syntax but will never modify the actual audio data.
+**loudgain** is a versatile ReplayGain 2.0 loudness normalizer, based on the EBU R128/ITU BS.1770 standard (-18 LUFS) and supports FLAC/Ogg/MP2/MP3/MP4/M4A/ALAC/Opus/ASF/WMA/WAV/WavPack/AIFF/APE audio files. It uses the well-known `mp3gain` commandline syntax but will never modify the actual audio data.
 
 Just what you ever wanted: The best of mp3gain, ReplayGain 2.0 and Linux combined. **Spread the word!**
 

From d6ae9c08ca0e88cc3caebd499185c26f9c1420da Mon Sep 17 00:00:00 2001
From: Moonbase59 <moonbase@quantentunnel.de>
Date: Sat, 7 Sep 2019 11:01:09 +0200
Subject: [PATCH 2/2] man page: Add APE to 's- i'

---
 docs/loudgain.1      |  2 +-
 docs/loudgain.1.html | 10 +++++-----
 docs/loudgain.1.md   |  2 +-
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/docs/loudgain.1 b/docs/loudgain.1
index 8f59138..93124b9 100644
--- a/docs/loudgain.1
+++ b/docs/loudgain.1
@@ -67,7 +67,7 @@ Delete ReplayGain tags from files\.
 .
 .TP
 \fB\-s i, \-\-tagmode=i\fR
-Write ReplayGain 2\.0 tags to files\. ID3v2 for MP2, MP3, WAV and AIFF; Vorbis Comments for FLAC, Ogg, Speex and Opus; iTunes\-type metadata for MP4/M4A; WMA tags for ASF/WMA; APEv2 tags for WavPack\.
+Write ReplayGain 2\.0 tags to files\. ID3v2 for MP2, MP3, WAV and AIFF; Vorbis Comments for FLAC, Ogg, Speex and Opus; iTunes\-type metadata for MP4/M4A; WMA tags for ASF/WMA; APEv2 tags for WavPack and APE\.
 .
 .TP
 \fB\-s e, \-\-tagmode=e\fR
diff --git a/docs/loudgain.1.md b/docs/loudgain.1.md
index 7a2ed0a..42082e5 100644
--- a/docs/loudgain.1.md
+++ b/docs/loudgain.1.md
@@ -57,7 +57,7 @@ Experimental, use with care: WAV (.wav), AIFF (.aiff, .aif, .snd).
 * `-s i, --tagmode=i`:
   Write ReplayGain 2.0 tags to files. ID3v2 for MP2, MP3, WAV and AIFF; Vorbis
   Comments for FLAC, Ogg, Speex and Opus; iTunes-type metadata for MP4/M4A;
-  WMA tags for ASF/WMA; APEv2 tags for WavPack.
+  WMA tags for ASF/WMA; APEv2 tags for WavPack and APE.
 
 * `-s e, --tagmode=e`:
   like '-s i', plus extra tags (reference, ranges).

